drivers/adc/ad4080/ad4080.h

```diff
@@ -216,6 +220,21 @@ enum ad4080_intf_ldo_pd {
 	AD4080_INTF_LDO_DISABLE,
 };
 
+/** AD4080 GPIO Function Select */
+enum ad4080_gpio_sel {
+	AD4080_GPIO_ADI_NSPI_SDO_DATA,
+	AD4080_GPIO_FIFO_FULL,
+	AD4080_GPIO_FIFO_READ_DONE,
+	AD4080_GPIO_FILTER_RESULT_READY,
+	AD4080_GPIO_HT_DETECT,
+	AD4080_GPIO_LT_DETECT,
+	AD4080_GPIO_STATUS_ALERT,
+	AD4080_GPIO_GPO_DATA,
+	AD4080_GPIO_FILTER_SYNC_INPUT,
+	AD4080_GPIO_EXT_EVENT_TRIGGER_FIFO,
+	AD4080_GPIO_CNV_INHIBIT_INPUT
+};
+
```

Comment:
I see these new definitions here but I do not see how these can be used... don't they need to be used in a driver API?

Reply:
That makes sense. I've been using the ad4080_read/write functions in my application along with these definitions. I'll make a generic function out of that for GPIO configurations, and make it part of this API.

Reply:
I've updated the driver with the API